apps/coscrad-frontend/src/components/resources/songs/song-detail.full-view.presenter.tsx

```diff
@@ -41,6 +69,26 @@ export const SongDetailFullViewPresenter = ({
                 <AudioClipPlayer audioUrl={audioURL} />
             </Box>
 
+            <Box>
```

Comment:
Note that we will want to move this to the base resource detail view soon.

Reply:
[Here](https://www.pivotaltracker.com/story/show/187571980) is a story for that.